lib/reducer.js

```diff
@@ -18,6 +18,17 @@ function set(state, ...props) {
   return Object.assign({}, state, ...props);
 }
 
+function insertAfterCurrentPosition(state, content) {
+    // TODO: make this work
+    return {
+        lines: state.lines,
+        cursor: {
+            from: state.cursor.from,
+            pos: state.cursor.pos + 1
+        }
+    }
+}
+
```

Comment:
@sjurba I think this is the main problem with this PR; this method doesn't add the `break` line _and_ it messes up the state object so badly that the program crashes. :see_no_evil: Please advise if you have any ideas on how to make it work.

Reply:
Fixed, resolving.